src/Raven.Studio/typescript/viewmodels/database/indexes/editIndex.ts

```diff
@@ -203,17 +203,27 @@ class editIndex extends viewModelBase {
         });
 
         this.effectiveSearchEngineText = ko.pureComputed(() => {
-            if (this.searchEngine()) {
-                return editIndex.formatEngine(this.searchEngine());
+            if (this.searchEngineConfiguration()) {
+                return editIndex.formatEngine(this.searchEngineConfiguration());
             }
             
             return this.inheritSearchEngineText();
         });
+        
+        this.searchEngineConfiguration.subscribe((engine: Raven.Client.Documents.Indexes.SearchEngineType) => {
+            let valueToUpdate: Raven.Client.Documents.Indexes.SearchEngineType = "Lucene";
+            
+            if ((engine && engine === "Corax") || (!engine && this.defaultSearchEngine() === "Corax")) {
```

Comment:
if (engine === "Corax" || (!engine && this.defaultSearchEngine() === "Corax"))

Reply:
done